core/modules/user/user.admin.inc

```diff
@@ -900,3 +620,311 @@ function user_admin_role_delete_confirm_submit($form, &$form_state) {
   $form_state['redirect'] = 'admin/config/people/roles';
 }
 
+/**
+ * Form builder; Configure user email settings for this site.
+ *
+ * @ingroup forms
+ */
+function user_settings_email($form, &$form_state) {
+  $config = config('system.core');
+  $mail_config = config('user.mail');
+  $form['email_title'] = array(
+    '#type' => 'item',
+    '#title' => t('Emails'),
+  );
+
+  $form['help'] = array(
+    '#markup' => '<p>' . t('Use this form to customize the emails sent by your website. Each template may use tokens to populate special values such as a user account name or the password reset link.') . '</p>',
+    '#description' => 'Help text to give context to the User emails form.',
```

Comment:
We can remove this `#description` line. It doesn't display anything to the user. If desired you could make it a code comment instead.